chain/stmgr/stmgr.go

```diff
@@ -842,15 +859,15 @@ func (sm *StateManager) setupGenesisActors(ctx context.Context) error {
 				return err
 			}
 
-			if s.StartEpoch != 0 {
+			if s.StartEpoch() != 0 {
 				return xerrors.New("genesis multisig doesn't start vesting at epoch 0!")
 			}
 
-			ot, f := totalsByEpoch[s.UnlockDuration]
+			ot, f := totalsByEpoch[s.UnlockDuration()]
 			if f {
-				totalsByEpoch[s.UnlockDuration] = big.Add(ot, s.InitialBalance)
+				totalsByEpoch[s.UnlockDuration()] = big.Add(ot, s.InitialBalance())
 			} else {
-				totalsByEpoch[s.UnlockDuration] = s.InitialBalance
+				totalsByEpoch[s.UnlockDuration()] = s.InitialBalance()
```

Comment:
@Stebalien This hackily aggregates all the multisigs in the genesis state by unlock duration, solely for the purpose of calculating circulating supply. If there are 3 multisigs with an unlock duration of 100 epochs, and initial balances of 200, 300, and 400, they can be considered as one multisig with unlock duration 100, and initial balance 900. 

I think this entire thing is unnecessary now, though...

Reply:
I see. It would be kind of nice to get rid of this, just so we can get rid of the actual functions (in case we need to change the behavior of multisigs in the future and, e.g., "unlock duration" no longer makes sense.